Sources/CropImage/CropImageView.swift

```diff
@@ -144,13 +144,13 @@ public struct CropImageView<Controls: View, CutHole: View>: View {
         if #available(iOS 16.0, macOS 13.0, *) {
```

Comment:
I don't think you need to change any `#if`s in this file. Rather, because visionOS 1.0 is born with `ImageRenderer` support, you can just add visionOS to this `#available` and let visionOS use the SwiftUI `ImageRenderer`:
```suggestion
        if #available(iOS 16.0, macOS 13.0, visionOS 1.0, *) {
```

Reply:
Please undo your changes on `#if`s in this file. There's no reason to make those back-compat code available to visionOS anymore.

Reply:
@laosb done